.gitignore

```diff
@@ -39,6 +39,7 @@ wp-tests-config.php
 /src/wp-includes/blocks/**/*.css
 /src/wp-includes/blocks/**/*.js
 /src/wp-includes/blocks/**/*.js.map
+/src/wp-content/themes/twentytwentyfive/style.min.css
```

Comment:
Should this ignore `/src/wp-content/themes/twentytwentytwo/style.min.css` too?

And could this/these belong after the `!/src/wp-content/themes/twentytwentyfive` line? The `node_modules` directories are in a later section.

Reply:
Good catch! Fixed in 7b87a85d98e37b9b60b69d587e50534e0aa98a1e.